annotations/static/annotations/js/annotators/text.js

```diff
@@ -546,14 +608,31 @@ AppellationCreator = {
             return '';
         },
         createAppellation: function () {
```

Comment:
isn't this whole function exactly the same as the other `createAppellation` one? This should be more dry.